### PR TITLE
Sync with latest changes from the scala version of the example

### LINKED ISSUE #31: Add OpenShift/Kubernetes deployment instructions
See https://github.com/playframework/play-scala-grpc-example/issues/30

### LINKED ISSUE #34: Upgrade docs to new play-grpc location
See https://github.com/playframework/play-scala-grpc-example/issues/36 for details.

### LINKED ISSUE #36: Sync with play-scala-grpc-example
Sync with https://github.com/playframework/play-scala-grpc-example, when it's feature complete.

### LINKED ISSUE #39: Port scala's docs
Once https://github.com/playframework/play-scala-grpc-example/pull/42 is merged, port the docs into this repo. The porting will require reviewing the contents and `@@snip` references.

### PR DESCRIPTION
**Pull request by ignasi35**
Fixes #36 
Fixes #31 
Fixes #34
Fixes #39

* changes in code
* changes in `keystore`
* changes in deployment settings

This PR makes the example app fully operation locally, on `minikube` and Central Park. 

This PR includes https://github.com/playframework/play-scala-grpc-example/pull/42 and https://github.com/playframework/play-scala-grpc-example/pull/46

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ build
 /.gradle
 /RUNNING_PID
 *.iml
+.envrc
```
Comment (marcospereira) [inline .gitignore]: 👀

**Review by marcospereira — APPROVED** [review]
2t408e

YAML good to me.

File: docs/src/main/paradox/openshift.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+service/play-java-grpc-example   ClusterIP   172.30.205.57   <none>        9000/TCP,9443/TCP   17h
+
+NAME                                                    DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/play-java-grpc-example-v1-0-snapshot   2         2         2            2           17h
+
+NAME                                                               DESIRED   CURRENT   READY     AGE
+replicaset.apps/play-java-grpc-example-v1-0-snapshot-5b77bd9849   2         2         2         16h
+
+NAME                                                     DOCKER REPO                                                                                         TAGS           UPDATED
+imagestream.image.openshift.io/play-java-grpc-example   docker-registry-default.centralpark.lightbend.com/play-java-grpc-example/play-java-grpc-example   1.0-SNAPSHOT   17 hours ago
```
Comment (octonato) [inline docs/src/main/paradox/openshift.md]: referencing centralpark, while Scala version has `mycluster.mycompany.com`

File: docs/src/main/paradox/openshift.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+NAME                                             HOST/PORT               PATH      SERVICES                  PORT      TERMINATION   WILDCARD
+route.route.openshift.io/play-java-grpc-route   myservice.example.org             play-java-grpc-example   http                    None
+```
+
+Test the application:
+
+```bash
+$ curl -H "Host: myservice.example.org" \
+        http://$OPENSHIFT_PROJECT.centralpark.lightbend.com
```
Comment (octonato) [inline docs/src/main/paradox/openshift.md]: same here

**Review by octonato — DISMISSED** [review]
LGTM, but sill referencing centralpark

File: docs/src/main/paradox/openshift.md (hunk 2 above)
Comment (ignasi35) [reply docs/src/main/paradox/openshift.md]: Oh, wow. Eagle Eyed Renato! 🦅

File: .gitignore (hunk 1 above)
Comment (ignasi35) [reply .gitignore]: Meh, OK. I'll move it out to my global ignores.

**Pull request merged**